### DISCUSSION TITLE
What is a Comlink?

### DISCUSSION THREAD

**Discussion by codewithcheese** [superfaceai/.github#11] [Q&A] [answered]
Hi, great idea. I see mentions of Comlink's on the website. What is a Comlink? Is that an in-house format or a standard that I am not aware of?

**Comment by martyndavies** [comment]
Hey @codewithcheese, thanks for the question.

Comlink is a domain-specific language we created to describe folks' different use cases with APIs. Think of it as a way of describing the capability of an API separate from the details for implementing that capability.

With Superface, you don't need to learn the Comlink language or write Comlink files to use us; our CLI handles all of that for you, and our SDK uses them to make your API use cases a reality.

If you're interested in digging more into Comlink, you can check out a full specification overview [in our documentation](https://superface.ai/docs/comlink/specification).

To get started with Superface, jump in with the CLI: [https://superface.ai/docs/introduction/quick-start](https://superface.ai/docs/introduction/quick-start)

**Reply by martyndavies** [reply, marked as answer]
@codewithcheese We've now increased the amount of explainers we have for Comlinks in our documentation, including this starter doc: [What are Comlinks?](https://superface.ai/docs/basics/what-are-comlinks). Just released today so I wanted to let you know about it.
